src/data/dataset.py

```diff
@@ -12,8 +27,12 @@ def __int__(self):
     def get(dataset: str) -> TimeSeries:
         if dataset not in Dataset.supported_datasets:
             raise ValueError(f"{dataset} not in {Dataset.supported_datasets}")
-
-        path = Dataset.path.format(dataset, dataset)
+        
+        if dataset in Dataset.baseline_datasets:
+            path = Dataset.path.format(dataset, dataset, "txt")
+        else:
+            path = Dataset.path.format("crypto", dataset, "csv")
+            
         series = TimeSeries.from_csv(path, header=None)
         series += 1e-9
         return series.univariate_component(0)  # TODO validate
```

Comment:
doing Dataset.get("BTC") should return a TimeSeries with only the BTC ticker, but this is not what happens [here](https://github.com/MarthyGarcia/IFT6759_B_H23/pull/10/files#diff-c6d5924ae6149f46ff72e8bf3087c02e5a871b0cd4aa2470d24a6f85996b81caL19).
It will return the first component due to this line here

Please add expected behavior